skimage/draw/draw.py

```diff
@@ -693,3 +693,57 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+
+def rectangle(origin, extent, shape=None):
+    """Generate coordinates of pixels within an nd-rectangle.
+
+    Parameters
+    ----------
+    origin : tuple
+        origin of the rectangle, (page,row,column,channel)
+    extent : tuple
+        extent (num_pages,num_rows,num_cols,num_chanels..) of the rectangle.
+    shape : tuple, optional
+        Image shape which is used to determine the maximum extent of output
+        pixel coordinates. This is useful for rectangles that exceed the image
+        size. If None, the bounds are determined from the bounds of
+        the rectangle.
+
+    Returns
+    -------
+    coords : (M, n) ndarray of int with the
```

Comment:
Consistency: "(M, N)". Also, this line should only contain the type, not the start of the description.